tests/js/test.js

```diff
@@ -200,6 +200,13 @@ describe('board.fen()', function () {
   });
 });
 
+describe('board.fen(showPromoted, countStarted)', function () {
+  it("it returns the current position in fen format, modified according to additional parameters", () => {
```

Comment:
For me the sentence _modified according to additional parameters_ seems not to sufficiently explain what these additional parameters are supposed to do.
How are promoted pieces indicated using `showPromoted`? After checking the source code, I guess using a `~` symbol after the corresponding piece.
I assume `countStarted` sets the no-progress counter manually, in the FEN.

Can you make the comment more explicit?  e.g.
```js
"it returns the current position in fen format. Promoted pieces are followed by the symbol '~' if `showPromoted` is true. The parameter `countStarted` overwrites the non-progress counter."
```